### ISSUE TITLE
Training module: ASL/LSQ video button

### ISSUE THREAD

**Issue by cherylhjli** [accessibility-in-action/platform#1871]
**Prerequisites**

- [x] I've [searched](https://github.com/accessibility-exchange/platform/issues) for existing issues

**Describe the bug**
The two sign language buttons are really confusing when next to each other - they look like duplicates. 

**Expected behavior**
Reduce the space between first sign language button and the question. Put the second sign language video under the answers.

**Comment by jobara** [comment]
@cherylhjli I agree that the placement isn't the best. I sort of left it like this though because our conventions in the site for forms were to have the video under the text and before the options. However, in our other cases it is a single video. This is something worth thinking about in conjunction with #1546

**Comment by cherylhjli** [comment]
@jobara as per our discussion, the conventions make sense, but unfortunately we have 2 videos instead of 1, which is what makes it confusing! I think this is a temporary solution until we can address it fully in #1546.
